docs/ref/models/fields.txt

```diff
@@ -1116,7 +1116,7 @@ Django also defines a set of fields that represent relations.
 ``ForeignKey``
 --------------
 
-.. class:: ForeignKey(othermodel, [**options])
+.. class:: ForeignKey(othermodel, on_delete, [**options])
```

Comment:
should we modify the methods' signature by moving it there? That's we'll have to do anyway when we will be removing the default althogether in 2.0.